
int32 mode
float64[] pgain
float64[] dgain

